libmscore/sym.cpp

```diff
@@ -2683,10 +2683,10 @@ const std::array<const char*, int(SymId::lastSym)+1> Sym::symUserNames = { {
       QT_TRANSLATE_NOOP("symUserNames", "No symbol"),
       "4-string tab clef",
       "6-string tab clef",
-      "11 large diesis down, 3\u00b0 down [46 EDO]",
-      "11 large diesis up, (11L), (sharp less 11M), 3\u00b0 up [46 EDO]",
-      "11 medium diesis down, 1\u00b0[17 31] 2\u00b046 down, 1/4-tone down",
-      "11 medium diesis up, (11M), 1\u00b0[17 31] 2\u00b046 up, 1/4-tone up",
+      QT_TRANSLATE_NOOP("symUserNames", "11 large diesis down, 3\u00b0 down [46 EDO]"),
+      QT_TRANSLATE_NOOP("symUserNames", "11 large diesis up, (11L), (sharp less 11M), 3\u00b0 up [46 EDO]"),
+      QT_TRANSLATE_NOOP("symUserNames", "11 medium diesis down, 1\u00b0[17 31] 2\u00b046 down, 1/4-tone down"),
+      QT_TRANSLATE_NOOP("symUserNames", "11 medium diesis up, (11M), 1\u00b0[17 31] 2\u00b046 up, 1/4-tone up"),
```

Comment:
the above 2 may fail to translate properly, due to the 2nd UTF char (for °) being followed by (hex) numbers